package.json

```diff
@@ -97,5 +100,10 @@
   "ava": {
     "verbose": true
   },
-  "typings": "./out/electron-builder.d.ts"
+  "typings": "./out/electron-builder.d.ts",
+  "config": {
+    "ghooks": {
```

Comment:
Opposite to precommit, ghooks correctly supports IDE (https://github.com/gtramontina/ghooks/issues/18) I have tested IntelliJ IDEA 2016.